doc/PRE_UPGRADE.md

```diff
@@ -1,2 +1,3 @@
-GoToSocial takes some time (from some seconds to some minutes) after each version upgrades to perform various things, like database migration or optimizations.  
-If your instance is not accessible, please be patient and **do not** restart it manually at the risk of breaking everything!
+GoToSocial takes some time (from some seconds to some hours) after each version upgrades to perform various things, like database migration or optimizations. 
+This takes place silently without any progress information after the yunohost upgrade process.
```

Comment:
it's not true, there are indeed progress information in the GTS logs, it's not that silent

Reply:
also, since the 0.18.0, when visiting the domain name of the app, a page is showed to warn that GTS is performing some maintenance tasks and so it's inaccessible
I also warned in [a special message](https://github.com/YunoHost-Apps/gotosocial_ynh/blob/master/doc/PRE_UPGRADE.d/0.18.0~ynh1.md) showed only when performing an update from a version before the 0.18.0 to the 0.18.0 or a greater version, that it will be exceptionally longer (with the example of my own instance)
so i definitively think it's not fair to say it's silent and that the info wasn't there

Reply:
Thank you. I did not see hints in the GTS logs during  my upgrade from 0.17.4 to 0.18.0.

I also assume a lot of rookie users to use yunohost, that do not mind the log files.

With silent, I've meant "no GUI message to the instance owner" (like nextcloud maintenance mode message). 

I wasn't aware that that kind of message has also arrived with 0.18.x.